static/app/utils/events.tsx

```diff
@@ -421,6 +421,7 @@ export function getAnalyticsDataForEvent(event?: Event | null): BaseEventAnalyti
     num_in_app_stack_frames: event ? getNumberOfInAppStackFrames(event) : 0,
     num_threads_with_names: event ? getNumberOfThreadsWithNames(event) : 0,
     event_platform: event?.platform,
+    event_runtime: event?.tags?.find(tag => tag.key === 'runtime')?.value ?? '',
```

Comment:
do we need to pass in the empty string if event_runtime is an optional param?

Reply:
Good point, I think we needed to ensure that keys always had a single type in the past but I don't think that's true anymore. Removed the fallback